File: keiner_lindarte_aro/instructor_henry/mis_programas/dart/5_clase_29_de_mayo/actividad.dart
import 'dart:io';

void main () {

  int? opcion;

do {

  print('Ingrese un numero entero:');
  String input = stdin.readLineSync()!;

  if (input.isNotEmpty && !input.contains(' ')){
    opcion = int.tryParse(input)!;
  } else {
    print('Ingrese un numero valido y sin espacios');
  }

  } while (opcion == null);

  if (opcion >= 1) {
    print('Su numero digitado es un numero positivo: $opcion');
  } else if (opcion <= -1) {
    print('Su numero digitado es un numero negativo: $opcion');
  } else {
    print('Su numero digitado es igual a cero: $opcion');
  }

}

File: keiner_lindarte_aro/instructor_henry/mis_programas/ruby/5_clase_29_de_mayo/actividad.rb
puts "Ingrese un numero entero:"
opcion = gets.chomp.to_i

if opcion >= 1 
    puts "Su numero digitado es un numero positivo: #{opcion}"
    elsif opcion <= -1
        puts "Su numero digitado es un numero negativo: #{opcion}"
    else
        puts "Su numero digitado es igual a cero: #{opcion}"
end